libsolidity/formal/Predicate.cpp

```diff
@@ -237,7 +237,7 @@ string Predicate::formatSummaryCall(
 	auto last = first + static_cast<int>(fun->parameters().size());
 	solAssert(first >= _args.begin() && first <= _args.end(), "");
 	solAssert(last >= _args.begin() && last <= _args.end(), "");
-	auto inTypes = FunctionType(*fun).parameterTypes();
+	auto inTypes = FunctionType(*fun).parameterTypesReplacingUserTypes();
```

Comment:
I feel that this function `parameterTypesReplacingUserTypes` does not belong in `Types.h`, but as a helper function somewhere here. What do you think?

Reply:
This would just be an applyMap or range-v3 transform using the function `underlyingType`.

Reply:
Ok!